Grid/parallelIO/IldgIO.h

```diff
@@ -744,17 +781,23 @@ class IldgReader : public GridLimeReader {
     FieldNormMetaData FieldNormMetaData_;
 
     // track what we read from file
-    int found_ildgFormat    =0;
-    int found_ildgLFN       =0;
-    int found_scidacChecksum=0;
-    int found_usqcdInfo     =0;
-    int found_ildgBinary =0;
-    int found_FieldMetaData =0;
+    int found_ildgFormat        =0;
+    int found_ildgLFN           =0;
+    int found_scidacChecksum    =0;
+    int found_usqcdInfo         =0;
+    int found_ildgBinary        =0;
+    int found_FieldMetaData     =0;
     int found_FieldNormMetaData =0;
     uint32_t nersc_csum;
     uint32_t scidac_csuma;
     uint32_t scidac_csumb;
 
+	// do we need to reconstruct the lattice from a reduced format?
```

Comment:
I'd suggest rephrasing this to not be a question, so it's clearer to the reader that you're not confused, but rather describing what the below variable does.